src/components/Navigation/Pagination/Pagination.module.css

```diff
@@ -5,7 +5,7 @@
 }
 
 .wrapper--disabled {
-  opacity: .35;
```

Comment:
Why would you do that?

Reply:
npx prettier did that

Reply:
I think it's wise to fix prettier in another PR and review it rules. This change conflicts with project codestyle

Reply:
I returned initial prettier config and css changes of prettier. Also created issue about it https://github.com/Telegram-Mini-Apps/TelegramUI/issues/35

Reply:
This PR still contains unnecessary changes, like file renaming and lines splitting

Reply:
Right, fixed that.